botCommands/debug.js

```diff
@@ -2,8 +2,32 @@ const { registerBotCommand } = require("../botEngine.js");
 
 const command = {
   regex: /(?<!\S)\/debug(?!\S)/,
-  cb: () =>
-    `Looks like this problem can be solved using a debugger: <https://developers.google.com/web/tools/chrome-devtools/javascript>`,
+  cb: ({ content }) => {
+    const query = content.match(/\B\/debug\s?(\S+)?/)[1];
+
+    const template = `Based on the description of your problem, you can get to the root of it using a debugger. Learning how to track down problems like this is an inevitable part of being a developer.`;
+
+    const defaultReply =
+      template +
+      `\n<https://en.wikipedia.org/wiki/Debugging>
+
+To get a helpful resource in javascript or ruby, run \`/debug js\` or \`/debug rb\`.`;
```

Comment:
Not in love with how we're mixing a template literal and traditional string concatenation here. Any reason why we can't just use a template literal here for the entire thing?